### ISSUE TITLE
Status entry should regain focus after URL shortening

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#182]
```
Steps that will reproduce the problem:
1. Paste a URL into the New Status entry
2. Press the URL shortening shortcut, Control+U
3. Press Enter

Expected output:
New status message uploaded to twitter

Real output:
Nothing, as the entry didn't gain focus automatically; consequently, the 
pressing of the Enter key did not send the status.

System:
Ubuntu 10.04 'Lucid Lynx'
Pino 0.2.4
```

Original issue reported on code.google.com by `wolte...@gmail.com` on 2 Apr 2010 at 6:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 2 Apr 2010 at 6:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 608b909bdb revision.
```

Original comment by `tro...@gmail.com` on 2 Apr 2010 at 7:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
So fast :)
```

Original comment by `wolte...@gmail.com` on 2 Apr 2010 at 11:12
